voidcrew/modules/cloning/code/computer_cloning.dm

```diff
@@ -0,0 +1,627 @@
+#define AUTOCLONING_MINIMAL_LEVEL 3
+
+/obj/machinery/computer/cloning
+	name = "cloning console"
+	desc = "Used to clone people and manage DNA."
+	icon_screen = "dna"
+	icon_keyboard = "med_key"
+	circuit = /obj/item/circuitboard/computer/cloning
+	req_access = list(ACCESS_GENETICS) //for modifying records
```

Comment:
We don't generally manage access on stuff here, do we want this to be access-locked?